src/Model/Stream/Csv.php

```diff
@@ -38,7 +38,7 @@ public function getContent(): string
 
     private function getStream(): WriteInterface
```

Comment:
Need to change return value to:
`?WriteInterface`

Reply:
I don't think so. `getStream()` always will return a `WriteInterface` instance. i.e. `$directory->openFile()` will always return a `WriteInterface` - or throw an exception otherwise. Thus this method will not ever return null (nor should it).